.mailmap

```diff
@@ -22,6 +23,8 @@ David Sheets <david.sheets@docker.com> <sheets@alum.mit.edu>
 Ian Campbell <ian.campbell@docker.com> <ijc25@users.noreply.github.com>
 Ian Campbell <ian.campbell@docker.com> <ijc@docker.com>
 Ian Campbell <ian.campbell@docker.com> <ijc@users.noreply.github.com>
+Isaac Rodman <isaac@eyz.us> <isaac.rodman@healthtrio.com>
+Jeff Wu <JeffWuBJ@users.noreply.github.com>
```

Comment:
Looks like the sign-off used;

```
Signed-off-by: JeffWuBJ <jeff.wu.junfei@gmail.com>
```

Reply:
yes but on his Github page he gives his name as Jeff Wu https://github.com/JeffWuBJ

Reply:
oh, yes, meant the e-mail address (edited my comment)

Reply:
ah yes